package.json

```diff
@@ -87,32 +85,19 @@
     "@types/ua-parser-js": "^0.7.39",
     "@types/uuid": "^3.4.13",
     "browserslist": "^4.24.4",
-    "browserstack-local": "^1.5.6",
-    "copyfiles": "^2.4.1",
     "dotenv-cli": "^8.0.0",
     "env-cmd": "^10.1.0",
     "expect": "^26.6.2",
-    "express": "^4.21.2",
     "husky": "^9.1.7",
     "next-transpile-modules": "^10.0.1",
-    "npm-force-resolutions": "0.0.3",
-    "npmignore": "^0.2.0",
-    "react-test-renderer": "^18.3.1",
     "rimraf": "^2.7.1",
-    "source-map-explorer": "^2.5.3",
-    "ttag-cli": "^1.10.18",
     "typescript": "^5.7.3"
   },
   "scripts": {
     "dev": "next dev",
-    "lint": "next lint",
```

Comment:
While I think about it, it probably makes sense to replace this with Biome. Will do later!